data/transactions/logic/assembler_test.go

```diff
@@ -345,6 +345,7 @@ itxna Logs 3
 const v6Nonsense = v5Nonsense + `
 itxn_next
 base64_decode URLEncoding
+json_ref JSONInt
```

Comment:
I think we need to call the constant JSONUInt.  Because it needs to fail if the value is negative, and making that clearer would be good.  Maybe even JSONUInt64, to be clear about the limits.